Sources_v3/StreamChatUI/Common Views/CurrentChatUserAvatarView.swift

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class CurrentChatUserAvatarView<ExtraData: UIExtraDataTypes>: AvatarView, AppearanceSetting {
+    // MARK: - Content
+    
+    public var currentUser: _CurrentChatUser<ExtraData.User>? {
+        didSet { updateContent() }
+    }
+    
+    open func updateContent() {
+        if let imageURL = currentUser?.imageURL {
+            imageView.setImage(from: imageURL)
+        } else {
+            imageView.image = nil
+        }
+    }
+    
+    // MARK: - Overrides
+    
+    override open var intrinsicContentSize: CGSize {
+        .init(width: 28, height: 28)
```

Comment:
Same comments here - this should be configurable + why 28x28? :)

Reply:
The first idea was to go with the **40x40** size defined by `AvatarView` but it in the UI it looks wild: the current user avatar doesn't fit the bar and has different left inset with channel avatars. 

That's was mainly the motivation to make the `CurrentChatUserAvatarView` smaller 🙂 

![Simulator Screen Shot - iPhone 11 - 2020-11-18 at 13 32 25](https://user-images.githubusercontent.com/12818985/99519144-86539f00-29a2-11eb-859c-6c1a14f715d6.png)

Reply:
Oh yeah, the designs have Android-height of 56pt, so we have to scale it down proportionally.

nit: Are you sure the bar margins are correct? I'd expect they should be aligned with the margins of the cells 🤔

Reply:
@evsaev See how the navbar margins match the cell margins in LLC sample app? 

![Screen Shot 2020-11-18 at 13 21 25](https://user-images.githubusercontent.com/6388510/99530086-214b7980-29a1-11eb-9ed4-a7613fb150b7.png)

Reply:
hm. Checking what's is different in the sample app

Reply:
@evsaev Did you find the issue?

Reply:
The navigation bar layout margins are the same on all the screens, doesn't matter what's in navigation item so that's probably the cell issue. Decided to get back to this later as we are a bit out of time now.